### DISCUSSION TITLE
Two Datasets as Different Task

### DISCUSSION THREAD

**Discussion by MagicHealer** [ContinualAI/avalanche#1151] [General]

**Comment by AndreaCossu** [comment]
Yes, please take a look at the [dataset_benchmark](https://avalanche-api.continualai.org/en/v0.2.1/generated/avalanche.benchmarks.generators.dataset_benchmark.html) generator. Just wrap each dataset into an `AvalancheDataset` and you are good to go!

**Reply by MagicHealer** [reply]
@AndreaCossu Thank you for this. May I also ask if you already have sample implementation of this? I am new to CL and if you have sample implementation using two datasets for two different tasks, it would be a great help. Thank you.
